cw/cw305/simple_capture_traces_batch.py

```diff
@@ -97,17 +98,34 @@ def run_batch_capture(capture_cfg, ot, ktp, scope):
                 f"actual: {actual_last_ciphertext}\n"
                 f"expected: {expected_last_ciphertext}"
             )
+            # Make sure to allocate sufficient memory for the storage segment array during the
```

Comment:
Would it make sense to add a reference to the CW issue here? newaetech/chipwhisperer#344

Reply:
Done.